src/filter_utils.cpp

```diff
@@ -45,7 +45,8 @@ void generate_label_indices(path input_data_path, path final_index_path_prefix,
 
         size_t number_of_label_points, dimension;
         diskann::get_bin_metadata(curr_label_input_data_path, number_of_label_points, dimension);
-        diskann::Index<T> index(diskann::Metric::L2, dimension, number_of_label_points, false, false);
+        diskann::Index<T> index(diskann::Metric::L2, dimension, number_of_label_points, nullptr, nullptr, 0, false,
```

Comment:
Should we assume L2?

Reply:
It was there to begin with as you can see. The only change in this PR is calling new ctor with updated args.